content/200-orm/800-more/450-migrating-to-prisma/03-migrate-from-mongoose.mdx

```diff
@@ -808,7 +808,7 @@ const createDraft = async (req, res) => {
 }
 ```
 
-Note that Prisma Client's nested write here saves the initial query where the `User` record is first retrieved by its `email`. That's because, with Prisma you can connect records in relations using any unique property.
+Note that Prisma Client's nested write here saves the initial query where the `User` record is first retrieved by its `email`. That's because, with Prisma ORM you can connect records in relations using any unique property.
```

Comment:
```suggestion
Note that Prisma Client's nested write here saves the initial query where the `User` record is first retrieved by its `email`. That's because, with Prisma Client you can connect records in relations using any unique property.
```